app/pods/components/video/template.hbs

```diff
@@ -1,7 +1,12 @@
 <div local-class="video">
-  <Swipeable @swipeLeft={{this.markDeleted}} @swipeRight={{if @video.new this.markLater this.markDeleted}} @iconLeft="trash" @iconRight={{if @video.new "bookmark" "trash"}}>
+  <Swipeable
+    @swipeLeft={{if @video.deleted this.markLater this.markDeleted}}
+    @swipeRight={{if @video.later this.markDeleted this.markLater}}
+    @iconLeft={{if @video.deleted "bookmark" "trash"}}
+    @iconRight={{if @video.later "trash" "bookmark"}}
+  >
     <div local-class="bar">
-      <button local-class="play-button" type="button" {{on "click" (if this.playing this.player.stop @play)}}>
+      <button local-class="play-button" type="button" {{on "click" (if this.playing this.player.stop @play)}} disabled={{@video.deleted}}>
```

Comment:
should probably do this more thoroughly, not sure if "deleting" a video while it's playing will actually stop playing it either now that I think of it